infra/modules/gcp-psoxy-bulk/main.tf

```diff
@@ -177,7 +177,7 @@ resource "google_cloudfunctions_function" "function" {
     var.path_to_config == null ? {} : yamldecode(file(var.path_to_config)),
     var.environment_variables,
     var.config_parameter_prefix == null ? {} : { PATH_TO_SHARED_CONFIG = var.config_parameter_prefix },
-    var.config_parameter_prefix == null ? {} : { PATH_TO_INSTANCE_CONFIG = "${var.config_parameter_prefix}${upper(local.instance_id)}_" },
+    var.config_parameter_prefix == null ? {} : { PATH_TO_INSTANCE_CONFIG = "${var.config_parameter_prefix}${replace(upper(var.instance_id), "-", "_")}_" },
```

Comment:
Detected this -same as rest function-.

For Jira it was failing, as the `PATH_TO_INSTANCE_CONFIG` was `PSOXY_JIRA-CLOUD_` instead of `PSOXY_JIRA_CLOUD_`, causing that secrets never been found as the prefix mismatch the name of the secret created. Example:
`psoxy_JIRA-CLOUD_OAUTH_REFRESH_TOKEN` -> `psoxy_JIRA_CLOUD_OAUTH_REFRESH_TOKEN`